FilterError.py: Change class and exception name

mohit2872 opened the pull request:
`FilterError.py`: Change class and exception name

This ensures that class name and file name are `FilterError`
Exception is now based on `LookupError`

Closes https://github.com/coala/coala/issues/4848

userzimmermann commented:
@mohit2872 And regarding _"Squash the commits"_: Read http://api.coala.io/en/latest/Developers/Git_Basics.html :) Especially http://api.coala.io/en/latest/Developers/Git_Basics.html#squashing-your-commits

mohit2872 commented:
I have squashed commits and removed the first pr #4855.
Is there anything else that needs to be done.

Regards
Mohit Gupta

On Sun, Nov 12, 2017 at 5:09 PM, Stefan Zimmermann <notifications@github.com
> wrote:

> @mohit2872 <https://github.com/mohit2872> And regarding *"Squash the
> commits"*: Read http://api.coala.io/en/latest/Developers/Git_Basics.html
> :) Especially http://api.coala.io/en/latest/Developers/Git_Basics.html#
> squashing-your-commits
>
> —
> You are receiving this because you were mentioned.
> Reply to this email directly, view it on GitHub
> <https://github.com/coala/coala/pull/4856#issuecomment-343730965>, or mute
> the thread
> <https://github.com/notifications/unsubscribe-auth/AQpqyMLa6x2DzalzfHtPHYbUwV1P3zbGks5s1tjsgaJpZM4Qa0yf>
> .
>

mohit2872 commented:
Sorry I didn't see the BearFilterError part. Now I have made that change as
well.

Regards
Mohit Gupta

On Sun, Nov 12, 2017 at 7:23 PM, Mohit Gupta <mg909@snu.edu.in> wrote:

> I have squashed commits and removed the first pr #4855.
> Is there anything else that needs to be done.
>
> Regards
> Mohit Gupta
>
> On Sun, Nov 12, 2017 at 5:09 PM, Stefan Zimmermann <
> notifications@github.com> wrote:
>
>> @mohit2872 <https://github.com/mohit2872> And regarding *"Squash the
>> commits"*: Read http://api.coala.io/en/latest/Developers/Git_Basics.html
>> :) Especially http://api.coala.io/en/latest/
>> Developers/Git_Basics.html#squashing-your-commits
>>
>> —
>> You are receiving this because you were mentioned.
>> Reply to this email directly, view it on GitHub
>> <https://github.com/coala/coala/pull/4856#issuecomment-343730965>, or mute
>> the thread
>> <https://github.com/notifications/unsubscribe-auth/AQpqyMLa6x2DzalzfHtPHYbUwV1P3zbGks5s1tjsgaJpZM4Qa0yf>
>> .
>>
>
>

shreyans800755 commented:
@mohit2872 
Use `git grep -n InvalidFilterException` to get all the occurrences of the exception. And rename all.

@userzimmermann IMO, `InvalidFilterError` is more self-explantory, but `BearFilterError` is also not bad.

mohit2872 commented:
I made changes to all files having "InvalidFilterException".
Thanks for helping.

Regards
Mohit Gupta

On Sun, Nov 12, 2017 at 8:52 PM, Shreyans Doshi <notifications@github.com>
wrote:

> @mohit2872 <https://github.com/mohit2872>
> Use git grep -n InvalidFilterException to get all the occurrences of the
> exception. And rename all.
>
> @userzimmermann <https://github.com/userzimmermann> IMO,
> InvalidFilterError is more self-explantory, but BearFilterError is also
> not bad.
>
> —
> You are receiving this because you were mentioned.
> Reply to this email directly, view it on GitHub
> <https://github.com/coala/coala/pull/4856#issuecomment-343744655>, or mute
> the thread
> <https://github.com/notifications/unsubscribe-auth/AQpqyF6mqA2OhjULHj3C8aVhmMkilD1vks5s1w1SgaJpZM4Qa0yf>
> .
>

shreyans800755 commented:
@mohit2872 
Why are you updating package files ? Please undo those changes.

mohit2872 commented:
ok will do it

On Sun, Nov 12, 2017 at 10:48 PM, Stefan Zimmermann <
notifications@github.com> wrote:

> *@userzimmermann* requested changes on this pull request.
>
> Hey @mohit2872 <https://github.com/mohit2872> :) According to #4856
> (review)
> <https://github.com/coala/coala/pull/4856#pullrequestreview-75961287> :
> Please revert changes to package.json
>
> And untrack package-lock.json
>
> —
> You are receiving this because you were mentioned.
> Reply to this email directly, view it on GitHub
> <https://github.com/coala/coala/pull/4856#pullrequestreview-75965186>, or mute
> the thread
> <https://github.com/notifications/unsubscribe-auth/AQpqyAdLX9zfjKWAJgp1F-I3D-MfEpkUks5s1yhTgaJpZM4Qa0yf>
> .
>

aashutoshrathi commented:
@mohit2872 Please fix continuous-integration/appveyor/pr. This PR important to us and we need to fix it soon.

RaiVaibhav commented:
@aashutoshrathi  `appveyor` is failing because of some other problem, discussed on gitter

mohit2872 commented:
Do I need to do anything?
On Wed, Nov 15, 2017 at 6:25 PM, Vaibhav Rai <notifications@github.com>
wrote:

> @aashutoshrathi <https://github.com/aashutoshrathi> appveyor is failing
> because of some other problem, discussed on gitter
>
> —
> You are receiving this because you were mentioned.
> Reply to this email directly, view it on GitHub
> <https://github.com/coala/coala/pull/4856#issuecomment-344584027>, or mute
> the thread
> <https://github.com/notifications/unsubscribe-auth/AQpqyH_YpmHAnfw2yZPsf4pKZP5CM3gtks5s2t9EgaJpZM4Qa0yf>
> .
>

mohit2872 commented:
I have been assigned #4864 so I will be working on it in meantime.

userzimmermann commented:
@mohit2872 I'm closing this since it's superseded by https://github.com/coala/coala/pull/4875 and https://github.com/coala/coala/pull/4910

cc @jayvdb

userzimmermann commented:
And now I'm pushing the right button to really close this PR :D
